source/partials/release_notes/_release-17-3-0.md.erb

```diff
@@ -0,0 +1,55 @@
+#### UI Enhancements 
+
+* <%= link_to_commit '320f2236c9eec058d5a8dd35b17e021e56dac44c', 'Enanched Job Status Accessibility with addition of symbols to signify stage and job statuses.' %>
+    <figure>
+      <%= image_tag 'releases/17.3.0/stage_with_symbols.png', alt: 'Stage and Task Symbols' %>
+    </figure>
+* <%= link_to_commit 'd95d94b16f5c1783dd9f5ff9e943ea32cd33d117', 'Improvements to console logs, tasks descriptions now read like a command-line rather than a XML dump.' %>
+* <%= link_to_commit 'cfc778edfcf6c4b0b557e31ce1dd2fa9804ca63c', 'Build status indicator moved from build summary to breadcrump toolbar.' %>
+* <%= link_to_commit '5478a5460cd4fcb1a7380eb0e4ab3fbbeacb4166', 'Agents page improvement for better identification of elastic agents by addition of icon related to the type of agent in use.' %>
+    <figure>
+      <%= image_tag 'releases/17.3.0/elastic_agent_icon.png', alt: 'Template Permissions' %>
+    </figure>
+
+
+#### API Enhancements
```

Comment:
Need a breaking changes section with the following —

"On GoCD servers installed on windows or if using the zip installers, logs will now be written to logs directory instead of the working directory"